src/components/modals/FeedbackPopUp.js

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Modal from "./Modal";
+import { openModal } from "../../actions/ModalActions";
+import Button from "../common/Button";
+
+const FeedbackPopUp = () => {
+  const dispatch = useDispatch();
+  const [showModal, setShowModal] = React.useState(false);
+  const propertyLayerActive = useSelector(
+    (state) => state.landOwnership.displayActive
+  );
+
+  const closeModal = () => {
+    setShowModal(false);
+    dispatch({
+      type: "CLOSE_MODAL",
+      payload: "feedbackPopUp",
+    });
+  };
+
+  const openFeedbackForm = () => {
+    closeModal();
+    dispatch(openModal("feedbackForm"));
+  };
+
+  const handleCheckboxChange = (e) => {
+    const unChecked = !e.target.checked;
+    setShowModal(unChecked);
+    localStorage.setItem("showModal", JSON.stringify(unChecked));
```

Comment:
I think this 'showModal' variable could be a bit more descriptive e.g. 'askForFeedback' since it was a bit confusing when reading (at first I thought it was controlling the current visibility state of the modal)

Reply:
Also, how often will this popup show if the checkbox isn't selected? From my understanding, it will appear (after 5 mins) every time the property layer is enabled for a user... but I might not have read the code properly!

If that's right, I could see that getting a bit annoying, especially if they have already submitted feedback. And it would likely lead to everyone selecting the checkbox.

I'm wondering if we should store the latest popup date and latest submission date too, so we can control how often we ask for feedback. Maybe something like once a fortnight if unanswered and once every 3 months if answered?

Reply:
Thanks taking the time to review both PRs btw 🙏

Reply:
thanks both!
@rogup's suggestions of two weeks when not submitted and 3months when submitted is a good one.
But I also agree that we don't necessarily want to put lots of effort in with so much on right now.
At the least it would be good to not annoy people for a while once they tell us how they are using it.

Are we talking less than to make this less annoying? Or more than half a day?

Reply:
Sorry @lin-d-hop, my head was stuck in something else. Half a day should be fine.